docs/running-in-production-dockerized.md

```diff
@@ -0,0 +1,88 @@
+# Running in Production Dockerized
+
+As the PrairieLearn source code is publicly-available, it's possible to run PrairieLearn on your own infrastructure. Running a single instance of the PrairieLearn server may be appropriate for tens or hundreds of total users, and a number of universities have done this successfully.
+
+## Getting started
+
+Follow the steps to [install PrairieLearn with local source code](./installingLocal.md). Then run this command in the root folder:
+```sh 
+docker-compose -f docker-compose-production.yml up
+```
+Then access PrairieLearn from port `3000`.
+
+## Configuration
+
+PrairieLearn can be configured by a `config.json` in the root of the repository.
+
+- First make the file `config.json` in your root repository.
+- Add the following line to `docker-compose-production.yml` under `volumes`:
+
+```sh
+- ./config.json:/PrairieLearn/config.json
+```
+
+The `config.json` file should contain appropriate overrides for the keys in [`lib/config.js`](`https://github.com/PrairieLearn/PrairieLearn/blob/master/lib/config.js`). At a minimum, you'll probably want to update the various `postgres*` options to point it at your database.
+
+## Reverse Proxy
+
+A reverse proxy can be implemented using something like [Apache](https://httpd.apache.org/docs/2.4/howto/reverse_proxy.html) or [NGINX](https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/).
+
+In `config.json` to configure your domain add:
+
+```json
+{
+"serverCanonicalHost": "https://yourdomain"
+}
+```

```

Comment:
This seems relevant to non-Docker installations as well. Let's move this to the primary "Running in production" docs.

Reply:
moved to running in production docs